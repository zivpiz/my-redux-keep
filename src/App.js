import React, {Component} from 'react';
import AllNotes from './Components/AllNotes';
import './style.css';

class App extends Component {
    render() {
        return (
            <div>
                <h1 className="ui top attached header">Your Notes</h1>
                <AllNotes/>
            </div>
        );
    }
}

export default App;
