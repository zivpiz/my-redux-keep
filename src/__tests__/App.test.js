import App from '../App';
import React from 'react';

describe("Test App Component Basic Rendering", ()=>{
    test('Renders appropriate snapshot', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
});


