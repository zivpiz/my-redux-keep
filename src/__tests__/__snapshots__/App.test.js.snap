// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App Comp Basic Rendering renders appropriate snapshot 1`] = `
<div>
  <h1
    className="ui top attached header"
  >
    Your Notes
  </h1>
  <AllNotes />
</div>
`;
